src/wp-includes/wp-db.php

```diff
@@ -2551,20 +2551,16 @@ protected function process_fields( $table, $data, $format ) {
 				}
 			}
 
+			if ( ! function_exists( '__' ) ) {
+				wp_load_translations_early();
+			}
```

Comment:
Same here. Notice the guards are removed.

Reply:
Removed `functions_exists()` in commit https://github.com/WordPress/wordpress-develop/pull/1897/commits/6f12dad2b7772316d6636af606d5af912cad3ade.